abci/example/kvstore/kvstore.go

```diff
@@ -13,9 +13,9 @@ import (
 	"time"
 
 	"github.com/cometbft/cometbft/abci/types"
+	"github.com/cometbft/cometbft/cmtdb"
```

Comment:
Why do we need the `cmt` prefix? If it's located within `cometbft/cometbft`

Reply:
I thought it would make it consistent with other parts of the code in which we import packages by adding the `cmt` prefix.

Reply:
Yeah, but you can always alias `cmtdb github.com/cometbft/cometbft/db` if you want.

Reply:
I'll make the changes